README.md

```diff
@@ -6,3 +6,20 @@ This is a monorepo containing two main packages:
 - `web-components`
 
 The `core` package is for bundling the two main packages into one for publishing. The `storybook` package is for the combined story files from each `*-components` package.
+
+# Running Build via Storybook
+
+1. `cd packages/web-components/`
+    1. `yarn build`
+    2. `yarn install`
```

Comment:
It seems strange to run `yarn build` before `yarn install`... is that really the order?

Reply:
You are correct updating README with yarn install prior to yarn build